tests/components/do_verify_email.test.jsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {shallow, configure} from 'enzyme';
+
+import Adapter from 'enzyme-adapter-react-16';
+
+import {verifyUserEmail as VerifyUserMail} from 'mattermost-redux/actions/users';
+
+import DoVerifyEmail from 'components/do_verify_email/do_verify_email.jsx';
+
+configure({adapter: new Adapter()});
+
+describe('components/DoVerifyEmail', () => {
+    const requiredProps = {
+        location: {
+            query: {
+                token: '',
+                email: ''
+            }
```

Comment:
Would recommend to add values here (e.g. ``{token: 'token', email: 'test@test.com'}``), so we could do assertions like:
- ``expect(requiredProps.actions.verifyUserEmail).toHaveBeenCalledWith('token');``
- ``expect(browserHistory.push).toHaveBeenCalledWith('/login?extra=verified&email=test4%40test.com');``